cvmfs/catalog.cc

```diff
@@ -408,6 +414,44 @@ void Catalog::DropDatabaseFileOwnership() {
 }
 
 
+Catalog::ExternalDataStatus Catalog::GetExternalDataLocked() const {
```

Comment:
In the rest of the code, we use `Locked` and `Unlocked` suffixes in reverse meaning, where `Unlocked` means "this routine doesn't acquire a lock".